app/scenes/channel_info/channel_info_container.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import {getChannelStats} from 'service/actions/channels';
```

Comment:
Do I need to call this through a view file app/actions/views instead of calling it directly from services?